frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -21,6 +21,7 @@ import type {
 
 import { CHART_STYLE } from "metabase/visualizations/echarts/cartesian/constants/style";
 
+import { parseNumberValue } from "metabase/components/TokenField";
```

Comment:
Wdyt about moving this util function to shared code? I haven't checked but there is a high chance this import breaks static viz becaue due to very limited runtime it is very sensitive to what you can import